docs/explanations/Cloud-Images-Overview.md

```diff
@@ -0,0 +1,85 @@
+# Ubuntu Cloud Images
+
+Canonical produces new Ubuntu produces new images regularly for various clouds. The Public Cloud team produces, registers, and publishes these images to the Amazon AWS EC2, Amazon AWS EKS, Google GCE, and Microsoft Azure. Other clouds and products, CPC works directly with partners to create a regular release process. These partners include: IBM VPC, IBM Classic, Oracle Cloud Infrastructure (OCI), Google GKE products, and others. CPC also owns the publication of "Premium" images to cloud marketplaces. This includes products such as Ubuntu Pro, Ubuntu Pro Fips, Anbox Cloud, and more. This document will help users understand the different places you can find official Canonical supported Ubuntu images, some of the basic terminology used throughout our docs, and the high level release plan.
+
+## Terminology
+
+* Image: A machine image. These can come in many formats: vmdk, qcow, and more. These may be booted as VMs or Baremetal (when supported by our partners)
+* Serial: A unique ID for an image in YYYYMMDD.INCREMENT format. Y=Year, M=Month, D=Day. example: 202010720. This corresponds to the date of the archive for the image build, not necessarily the date an image was registered or published. Example: An image is publised to Microsoft Azure as 20220815. However, the ingestion pipeline takes 24 hours, and it is made available on 20220816. Thus, image creation date, as stated in the cloud, may be different than the version serial.
+* Registration: The act of taking an image, and making it available for use in the cloud. For example, the act of taking a vmdk, uploading, and importing it into ec2 as an ami
+* Publication: the act of taking a machine image and making it publicly available. This may involve a marketplace, or it may involve a flag in the permissions of the image itself.
+* Daily: a build that can occur on a daily basis. Basic testing only provided
+* Release: a build that occurs when a specific set of packages, the "release package set", has changed. This is a slower cadence, normally at longest the kernel SRU timeframe.
```

Comment:
I would leave the term "release package set" out here. Instead I would say something like:

`a publication that occurs when a boot critical package (eg. kernel, grub, cloud-init or image specific packages) has changed.